stripe/src/main/java/com/stripe/android/model/StripeFileParams.kt

```diff
@@ -0,0 +1,73 @@
+package com.stripe.android.model
+
+import android.os.Parcelable
+import java.io.File
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * [Create a file](https://stripe.com/docs/api/files/create)
+ *
+ * To upload a file to Stripe, you’ll need to send a request of type `multipart/form-data`.
+ * The request should contain the file you would like to upload, as well as the parameters for
+ * creating a file.
+ */
+internal data class StripeFileParams(
```

Comment:
Did you intend to mark this with `@Parcelize` too?

Reply:
I don't need it to be `@Parcelize` for now